packages/manager/apps/public-cloud/src/assets/theme/_sidebar.less

```diff
@@ -22,6 +49,13 @@ cloud-sidebar { /* stylelint-disable-line */
     will-change: transform;
     transition: transform ease-in-out 500ms;
 
+    oui-clipboard { /* stylelint-disable-line */
```

Comment:
Stylelint prevents creating styles for tags ? Maybe we should disable this rule then

Reply:
Yep :)